x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -288,6 +288,7 @@ static RoleDescriptor kibanaSystem(String name) {
                         "read",
                         "index",
                         "delete",
+                        "manage",
```

Comment:
The comment on L284 should be removed because it doesn't add any information, and it now does not match the code.

Reply:
Removed the commented line